.github/workflows/build.yml

```diff
@@ -0,0 +1,53 @@
+on:
+  push:
+    branches:
+      - "graal-build"
+jobs:
+  build:
+    runs-on: "ubuntu-20.04"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.m2/repository
+            ~/.gitlibs
+            ~/graalvm-ce-java17-22.0.0.2
+          key: ${{ runner.os }}-maven-${{ hashFiles('deps.edn') }}
+      - name: Download GraalVM
```

Comment:
I use this action to install graalvm: https://github.com/marketplace/actions/setup-graalvm-environment

https://github.com/avelino/moclojer/blob/598d8defe8d8e18b28f5d1b63b3afd3d60584792/.github/workflows/tests.yml#L10-L17